src/main.py

```diff
@@ -35,38 +35,9 @@ def main():
 
     # create the empty json file for images
     build_context = os.getenv("BUILD_CONTEXT")
```

Comment:
Nice job reducing lines in this main function! I personally think this makes it a bit more readable, though we just need to ensure that the functionalized forms are well documented

Reply:
Ack. Will add more details in docstrings.